### DISCUSSION TITLE
Team Fortress 2 Game Support

### DISCUSSION THREAD

**Discussion by sak0a** [nillerusr/source-engine#309] [General]
Hey guys,

1. Question:
Would it be possible to build Team Fortress 2 somehow?
(I know there is much work todo to port it)
2. Question
Is VAC a problem with the builds from this engine? (So is playing on Servers with VAC working?)

It would be big project but i'm playing that game for so long that i want it with 64 bit support to work on the newest MacOS Versions.

Thanks for any answers

**Comment by maksw2** [comment]
this fork was based on the tf2 leak, so go figure

**Comment by tinspin** [comment]
Can this be merged? https://github.com/OthmanAba/TeamFortress2

Or is it the same, why is there no tf2 build target?

**Reply by ItzVladik** [reply]
try this (is this TF2 2008 ver) -  https://github.com/ItzVladik/source-engine/tree/tf_port

**Reply by tinspin** [reply]
This is just a fork of this one, how do you build TF2 with this?

**Reply by maksw2** [reply]
same way, different --build-games parameter.
https://github.com/ItzVladik/source-engine/blob/0d9bc9ca43fa7c4c47de3efd54f73db488afee9e/.github/workflows/build.yml#L59

**Reply by tinspin** [reply]
So you changed ./waf.bat configure -T debug to ./waf.bat configure -T release --build-games=tf_port --prefix=bin

Why is there no tf_port folder? cstrike and portal have their own folders?

Wait those are made upon compiling?

So could one run python3 ./waf configure -T release --prefix=tf_port --build-games=tf_port --disable-warns?

Aha, gonna try this later!

**Comment by tinspin** [comment]
What about now: https://github.com/ValveSoftware/source-sdk-2013
